### ISSUE TITLE
cell summary file not found

### ISSUE THREAD

**Issue by apple5413** [sdw95927/Ceograph#1]
Hello, I am a medical student from China. I greatly benefited from your article "Deep learning of cell spatial organizations identifies clinically relevant insights in tissue images" and would like to learn from the code on your GitHub. However, I encountered an error when running ceograph_training_and_inference.ipynb to build the Torch graph dataset, stating "cell summary file not found." I would like to inquire how the cell summary file is generated. Is it obtained through the HD-Staining model?

**Comment by sdw95927** [comment]
Yes, this is obtained from the HD-Staining model and here is the online port for HD-Staining: https://lce.biohpc.swmed.edu/maskrcnn/analysis.php

**Comment by apple5413** [comment]
I appreciate the provided website, which allows me to upload images online and obtain the image mask and cell summary file, making it very convenient. However, it takes approximately 100 to 200 seconds to obtain results when uploading a 512*512 pixel image, which is relatively time-consuming. I wonder if it is convenient for you to share your HD-staining model along with the relevant usage code. This way, I can run the code on my local computer to obtain results. My research focuses on bladder cancer, so if you could share your model and code, I would be extremely grateful.

**Comment by sdw95927** [comment]
> I appreciate the provided website, which allows me to upload images online and obtain the image mask and cell summary file, making it very convenient. However, it takes approximately 100 to 200 seconds to obtain results when uploading a 512*512 pixel image, which is relatively time-consuming. I wonder if it is convenient for you to share your HD-staining model along with the relevant usage code. This way, I can run the code on my local computer to obtain results. My research focuses on bladder cancer, so if you could share your model and code, I would be extremely grateful.

Could you try this: https://github.com/impromptuRong/hd_wsi, this repo should contain the models and scripts. It just uses another model a little bit different from HD-Staining, but should have similar performance :)
